src/app.tsx

```diff
@@ -1,9 +1,23 @@
 import React, { useEffect } from 'react';
+import * as Sentry from '@sentry/react';
+import { BrowserTracing } from '@sentry/tracing';
 import StoreProvider from './store/store-provider';
 import { hentFnrFraUrl } from './util/url-utils';
 import { PersonflatePage } from './page/personflate';
 import { synkroniserManuellStatusMedDkif } from './api/api';
 
+if (process.env.NODE_ENV && process.env.NODE_ENV !== 'development') {
```

Comment:
Unngå å kjøre rapportere når man kjører lokalt

Reply:
Det er også en "erMock" funksjon som funker for å sjekke om man er lokalt

Reply:
Eller det kommer snart en erMock funksjon: https://github.com/navikt/veilarbpersonflatefs/pull/250/files